src/components/ScenarioParameters/ScenarioParameters.js

```diff
@@ -374,6 +371,8 @@ ScenarioParameters.propTypes = {
   addDatasetToStore: PropTypes.func.isRequired,
   updateAndLaunchScenario: PropTypes.func.isRequired,
   launchScenario: PropTypes.func.isRequired,
+  setAccordionSummaryExpanded: PropTypes.func.isRequired,
```

Comment:
Can we rename `setAccordionSummaryExpanded` to `onChangeAccordionSummaryExpanded` ? (with the "on" prefix for callbacks)